.github/stale.yml

```diff
@@ -0,0 +1,29 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 90
+
+# Number of days of inactivity before a stale Issue or Pull Request is closed
+daysUntilClose: 7
```

Comment:
This is 7 days after it has been marked as stale, so a total of 97 days, but we could up this to 14?